### DISCUSSION TITLE
installer created config.php vs project config.distrib.php

### DISCUSSION THREAD

**Discussion by dalers** [timschofield/webERP#253] [Problems / Bugs]
config.distrib.php has more explanations and additional variables compared to config.php created by the installer, also there seem to be some differences in variables:

- $host in config.distrib.php vs $Host in config.php
- $DefaultDatabase in config.distrib.php vs $DefaultCompany in config.php
- $CompanyList[] array in config.distrib.php vs no list in config.php

I also suspect some variables have been deprecated or are now redundant.

Would it be  possible for config.php to essentially be an edited version of config.distrib.php, so the comments, variables, order, etc. are the same?

**Comment by timschofield** [comment]
I agree Dale, I have converted this discussion to an issue to remind me

Tim
